default_data.json

```diff
@@ -25420,6 +25420,10 @@
                 },
                 {
                     "company_name": "Futurewei",
+                    "end_date": "2019-June-14"
```

Comment:
Looks like you have incorrect format. It must be 'Jun' instead of 'June'.

Reply:
Sorry, I missed your comment until now. Pushed up an update with the fixed name.